charts/event-handler/README.md

```diff
@@ -0,0 +1,257 @@
+
+```yaml
+---
+kind: role
+version: v5
+metadata:
+  name: teleport-plugin-event-handler
+spec:
+  allow:
+    logins:
+    - teleport-plugin-event-handler
+    rules:
+    - resources:
+      - event
+      verbs:
+      - list
+      - read
+  options:
+    forward_agent: false
+    max_session_ttl: 8760h0m0s
+    port_forwarding: false
+---
+kind: user
+version: v2
+metadata:
+  name: teleport-plugin-event-handler
+spec:
+  roles:
+    - teleport-plugin-event-handler
+```
+
+You can either create the user and the roles by putting the YAML above into a file and issuing the following command  (you must be logged in with `tsh`):
+
+```code
+tctl create user.yaml
+```
+
+or by navigating to the Teleport Web UI under `https://<yourserver>/web/users` and `https://<yourserver>/web/roles` respectively. You'll also need to create a password for the user by either clicking `Options/Reset password...` under `https://<yourserver>/web/users` on the UI or issuing `tctl users reset teleport-plugin-event-handler` in the command line.
+
+The next step is to create an identity file, which contains a private/public key pair and a certificate that'll identify us as the user above. To do this, log in with the newly created credentials and issue a new certificate (525600 and 8760 are both roughly a year in minutes and hours respectively):
+
+```code
+tsh login --proxy=proxy.example.com --auth local --user teleport-plugin-event-handler --ttl 525600
+```
+
+```code
+tctl auth sign --user teleport-plugin-event-handler --ttl 8760h --out teleport-plugin-event-handler-identity
+```
+
+Alternatively, you can execute the command above on one of the `auth` instances/pods.
+
+The last step is to create the secret. The following command will create a Kubernetes secret with the name `teleport-plugin-event-handler-identity` with the key `auth_id` in it holding the contents of the file `teleport-plugin-event-handler-identity`:
+
+```code
+kubectl create secret generic teleport-plugin-event-handler-identity --from-file=auth_id=teleport-plugin-event-handler-identity
+```
+
+### Mounting Fluentd client certificate
+
+See the [plugin's documentation](../../event-handler/README.md#mtls_advanced) about how to generate the certificates using fluentd's CA certificate and private key.
+
+Once the files `client.key` and `client.crt` were created successfully, the following command can be used to create a new secret (`ca.crt` is also included since we'll need it to verify we are connecting to the right fluentd):
+
+```code
+kubectl create secret generic teleport-plugin-event-handler-client-tls --from-file="ca.crt=ca.crt,client.key=client.key,client.crt=client.crt"
+```
+
+### Storage
+
+The event-handler plugin stores it's current state on the disk, so to avoid accidentally sending any events twice, it's a good idea to create a PersistentVolumeClaim to ensure those files are retained:
```

Comment:
Wouldn't it make more sense to add built-in support for the persistentvolume to the Helm chart itself ?

Reply:
Yeah, I was also thinking about that. I'm not sure how much people would want to customize it though (eg. is it enough to only expose `storageClassName` and maybe `requests.storage`?)

Reply:
Lots of charts do both, basically. Provide the generic mounts/volumes options, and also the shorter, just gimme xGB of storageclass y. All depending on which values you chose to use.